Can't verify contracts (Cannot find module 'solc')

### Environment

* Operating System: Linux
* Blockscout Version/branch: https://github.com/bobanetwork/blockscout/tree/alt-l1s

### Steps to reproduce

We use the `Dockerfile` in the docker folder to build the image, but it can't find `solc` package when we try to verify the contracts using the `@nomiclabs/hardhat-etherscan` package.

The logs from the image:
```
2022-08-03T04:52:47.115 application=que [info] [Que] Queued new Job for Explorer.SmartContract.Solidity.PublisherWorker
2022-08-03T04:52:47.115 application=que [info] [Que] Starting Job # 10 with Explorer.SmartContract.Solidity.PublisherWorker
2022-08-03T04:52:47.115 application=plug request_id=Fwe8nEebMkONDJwAFTWh [info] Sent 200 in 4ms
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'solc'
Require stack:
- /app/lib/explorer-0.0.1/priv/compile_solc_standard_json_input.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/app/lib/explorer-0.0.1/priv/compile_solc_standard_json_input.js:6:12)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/lib/explorer-0.0.1/priv/compile_solc_standard_json_input.js'
  ]
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,7 @@
 
 ### Fixes
 - [#5797](https://github.com/blockscout/blockscout/pull/5797) - Fix flickering token tooltip
+- [#5798](https://github.com/blockscout/blockscout/pull/5798) - Copy explorer node_modules to result image
 
 ### Chore
 - [#5796](https://github.com/blockscout/blockscout/pull/5796) - Add job for e2e tests on every push to master + fix job "Merge 'master' to specific branch after release"

--- a/docker/Dockerfile
+++ b/docker/Dockerfile
@@ -85,5 +85,6 @@ FROM bitwalker/alpine-elixir-phoenix:1.13
 WORKDIR /app
 
 COPY --from=builder /opt/release/blockscout .
+COPY --from=builder /app/apps/explorer/node_modules ./node_modules
 
 

